src/components/Tabs/TabsComponent.tsx

```diff
@@ -53,6 +69,33 @@ export default function TabsComponent({ mode, gameData, setGameData }: Props) {
                 <p>{gameData.description}</p>
               )}
             </TabPanel>
+            {gameData.videoTrailer && gameData.videoTrailer !== "" ? (
+              <TabPanel className="flex flex-col justify-center">
+                <ReactPlayer
+                  url={gameData.videoTrailer}
+                  controls={true}
+                  width={1300}
+                  height={731}
+                />
+                {mode === "edit" && (
+                  <Flex className="flex-row">
+                    <EditVideoTrailer gameData={gameData} />
+                    <DeleteVideoTrailer gameData={gameData} />
+                  </Flex>
+                )}
+              </TabPanel>
+            ) : null}
+            {(gameData.videoTrailer === undefined ||
```

Comment:
If AddVideoTrailer and EditVideoTrailer are the same component we wouldn't need this section (lines 88 to 94) either which would make the code more readable, you'd probably have to add another conditional statement at line 80 tho